html/browsers/windows/browsing-context-names/browsing-context-choose-parent-001.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML Test: Choose browsing context - '_parent'</title>
+<link rel="author" title="Intel" href="http://www.intel.com/">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<iframe id="embedded" name="parentWin" style="display:none"></iframe>
+
+<script>
+async_test(t => {
+  window.addEventListener('message', t.step_func_done(e => {
+    assert_equals(e.data.name, 'parentWin');
+  }));
+  window.frames['embedded'].src = 'resources/parent-iframe-1.html';
```

Comment:
I get confused about the intent here, if it's trying to set `src` on the iframe or on its `WindowProxy`. In Safari, if the iframe happens to not have a `name` attribute, this would return the `WindowProxy`. I think that is a bug per https://html.spec.whatwg.org/#named-access-on-the-window-object - but testing that should be separate.

Here it would be clearer to just use `document.getElementById('embedded')`.